README.md

```diff
@@ -51,4 +51,42 @@ npm run server-dev
 
 [Trello](https://trello.com/b/yqDl65OT/rfp2310-fcp-quiddich-queue-handl)
 
+## Contributing Work
 
+Every feature will be logged as a trello ticket
+Create a new branch for every feature
+
+New branches will follow the naming scheme:
+twoletterprefix-featurename
+
+The prefixes are as follows:
+pd- for Product Details
+rr- for Ratings and Reviews
+qa- for Questions and Answers
+ri- for Related Items and Outfit Creation
+
+Always branch from main with the latest changes:
+    - pull from origin main
+    - branch from local main
+
+To submit a PR:
+    - assure all changed files have been committed into
+    your (local) branch
+    - push branch into origin:
+        - git push origin branchname:branchname
+    - Navigate to branch in github and click create PR
+        - Or use the automatic create PR button
+    - The title of the PR should match the ticket title and include
+the ticket/card number:
+        - ie:
+          #12: [S] Create global.css file and import into React
+
+## CSS rules
```

Comment:
Can you include sticking to rem units instead of px for sizing in css

Reply:
Yes